app/src/main/java/nl/hogeschoolrotterdam/projectb/LocationEditActivity.java

```diff
@@ -0,0 +1,152 @@
+
+package nl.hogeschoolrotterdam.projectb;
+
+import android.app.Activity;
+import android.content.Intent;
+import android.location.Address;
+import android.location.Geocoder;
+import android.os.Bundle;
+import android.util.Log;
+import android.view.KeyEvent;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.view.inputmethod.EditorInfo;
+import android.widget.Button;
+import android.widget.EditText;
+import android.widget.TextView;
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+import androidx.appcompat.app.AppCompatActivity;
+import androidx.fragment.app.Fragment;
+import com.google.android.gms.maps.*;
+import com.google.android.gms.maps.model.CameraPosition;
+import com.google.android.gms.maps.model.LatLng;
+import com.google.android.gms.maps.model.Marker;
+import com.google.android.gms.maps.model.MarkerOptions;
+import nl.hogeschoolrotterdam.projectb.MainActivity;
+import nl.hogeschoolrotterdam.projectb.MemoryDetailActivity;
+import nl.hogeschoolrotterdam.projectb.R;
+import nl.hogeschoolrotterdam.projectb.data.Database;
+import nl.hogeschoolrotterdam.projectb.data.Memory;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Created by maartendegoede on 20/03/2019.
```

Comment:
I did not create this class. You may change this to your name :)
Also: It's 24/04/2019

Reply:
done